tests/docs/test_docs.py

```diff
@@ -13,11 +13,13 @@ def test_html(self):
         doctrees_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['doctrees'])
         html_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['html'])
 
-        check = subprocess.call(["sphinx-build", "-nW", "-b", "html", "-d",
-            "{0!s}".format((doctrees_path)) ,
-            "{0!s}".format((self.path_to_docs)),
-            "{0!s}".format((html_path))])
-        assert check == 0
+        check = subprocess.Popen(("sphinx-build", "-nW", "-b", "html", "-d",
+            "{0!s}".format(doctrees_path),
+            "{0!s}".format(self.path_to_docs),
+            "{0!s}".format(html_path)),
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = check.communicate()
+        assert check == 0, 'Test failed with stdout:\n{}\n and stderr:\n{}\n'.format(out,err)
```

Comment:
personally, I find this to be more difficult to sort through 
![image](https://cloud.githubusercontent.com/assets/6361812/18758020/19c56052-80ab-11e6-915a-d4f9e2a1a957.png)

The previous version is syntax highlighted in travis, so I would advocate for that. Th

![image](https://cloud.githubusercontent.com/assets/6361812/18758086/739d33b6-80ab-11e6-847a-65e0198b159f.png)

Thoughts @grosenkj , @rowanc1 ?